third_party/pcsc-lite/naclport/server_clients_management/src/permissions_checking/user-prompting-checker.js

```diff
@@ -148,6 +150,9 @@ UserPromptingChecker.prototype.check = function(clientOrigin) {
       },
       this);
 
+  delete goog.global[`googleSmartCard_resolveModalDialog${GSC.PopupOpener.lastUsedPopupId}`];
```

Comment:
This deletion should live under /popup-window/ - it shouldn't be that every callsite needs to take care of this cleanup. Also it's technically not fully precise: in case there are two dialogs shown at the same time, `lastUsedPopupId` will be "2", and the first dialog's variables will stay as a garbage.

Reply:
you mean because it's not being deleted per popup?

Reply:
Ah, that's another bug indeed. But my comment was referring to the location of this code: the right abstraction layer for this would be the common/js/popup-window/ folder.